bpy_addon_build/build_context/hook_definitions.py

```diff
@@ -0,0 +1,160 @@
+from typing import Optional, Union
+from bpy_addon_build.api import BpyError, BpyWarning
+from rich.console import Console
+
+from bpy_addon_build.build_context import BuildContext
+
+
+def build_action_prebuild(ctx: BuildContext, action: str, console: Console) -> None:
+    """
+    Runs an action's pre_build function
+
+    ctx: Build context
+    action: string representing the action name
+    console: Console from Rich
+
+    Returns:
+        None
+    """
+    if ctx.config.build_actions is None and len(ctx.cli.actions):
+        print("Actions must be defined to use them!")
+        return
+    if action in ctx.cli.actions:
+        if action not in ctx.api.action_mods:
+            print("Action not in API!")
+            return
+        if hasattr(ctx.api.action_mods[action], "prebuild"):
```

Comment:
```suggestion
        if hasattr(ctx.api.action_mods[action], "pre_build"):
```

Per your example file below, I think this needs to be `pre_build` instead of "prebuild", as it doesn't print "HELLO"